dataland-frontend/tests/e2e/fixtures/custom_mocks/StoredDataRequestsFaker.ts

```diff
@@ -0,0 +1,131 @@
+import { RequestStatus, type StoredDataRequest } from "@clients/communitymanager";
+import { DataTypeEnum } from "@clients/backend";
+import { generateInt } from "@e2e/fixtures/common/NumberFixtures";
+import { generateReportingPeriod } from "@e2e/fixtures/common/ReportingPeriodFixtures";
+import { generateStoredDataRequestMessage } from "@e2e/fixtures/custom_mocks/StoredDataRequestMessageFaker";
+import { faker } from "@faker-js/faker";
+import { generateArray, pickOneElement } from "@e2e/fixtures/FixtureUtils";
+
+/**
+ * Creates a list of stored data requests
+ * @returns The list of request
+ */
+export function generateStoredDataRequests(): StoredDataRequest[] {
+  const storedDataRequests = [];
+  storedDataRequests.push(generateStoredDataRequest());
+  storedDataRequests.push(
+    manipulateFixtureToHaveDataType(
```

Comment:
i would rather create one manipulator function with 4 optional arguments that manipulate the fixture, if the input is non-null.
then you avoid the function depth present here

Reply:
will do (Y)

Reply:
done